README.md

```diff
@@ -44,23 +44,24 @@ To run the application:
 yarn start
 ```
 
-**Note that some [configuration](#configuration) may be necessary to run the service in isolation**
+**Note that some [configuration](#environment-variables) may be necessary to run the service in isolation**
 
-## Testing
-
-To run all tests:
-```
-yarn test
-```
-
-## Configuration
+## Environment Variables
 
 The following environment variables can be configured.
 
 | Name | Description | Required |
 | --- | --- | --- |
 | `EQ_SCHEMA_VALIDATOR_URL` | The URL of the schema validation service. See [Running with Docker Compose](#running-with-docker-compose). | Yes |
-| `EQ_AUTHOR_API_URL` | The URL of the GraphQL API server. If not supplied, mock API will be used. | Yes |
+| `EQ_AUTHOR_API_URL` | The URL of the GraphQL API server | Yes |
+| `EQ_PUBLISHER_VERSION` | The current Publisher version | Yes |
```

Comment:
Maybe add a note saying this is what gets reported on the `/status` endpoint?

Reply:
Think you can change it to not required as well if it only gets used in the pipeline.

Reply:
@SamGodwin2 I was going to put no for required but left it as yes in the end because I found this bit of code which makes it look like it is now required:https://github.com/ONSdigital/eq-publisher/blob/8814c531f2cc6dd278fde9deeeceb215c2a83c8b/src/main.js#L21-L23

I could be wrong though?

@samiwel Just checking, do you mean that if you have the service running you can find out which version of publisher it is if you go to http://localhost:9000/status, and that it is this variable that the version information is coming from? Is that what you want me to make a note about?

Reply:
Ive pushed a new version with what I think are the correct changes

Reply:
Ah sorry I was talking about the `EQ_PUBLISHER_VERSION` variable, don't think that one is expressly needed anywhere.

Reply:
Ah yeah I've just had a look and I think you're right. Ive made the change and pushed up again